projects/packages/videopress/src/class-block-editor-content.php

```diff
@@ -21,31 +21,67 @@ public static function init() {
 			return;
 		}
 
-		add_filter( 'default_content', array( static::class, 'video_block_by_guid' ), 10, 2 );
+		add_shortcode( 'jetpack_videopress', array( static::class, 'videopress_block_shortcode' ) );
+		add_filter( 'default_content', array( static::class, 'videopress_block_by_guid' ), 10, 2 );
 	}
 
 	/**
-	 * Generates a video block content with the given guid
+	 * VideoPress embed shortcode
+	 *
+	 * Example use:
+	 * [jetpack_videopress guid=tLvEwHYZ width=560 height=315]
+	 *
+	 * @param array $atts Shortcode attributes.
+	 *
+	 * @return string html
+	 */
+	public static function videopress_block_shortcode( $atts ) {
+		$atts = shortcode_atts(
+			array(
+				'guid'   => '', // string.
+				'width'  => 560, // int.
+				'height' => 315, // int.
+			),
+			$atts,
+			'jetpack_videopress'
+		);
+
+		$guid = sanitize_text_field( wp_unslash( $atts['guid'] ) );
+
+		if ( empty( $guid ) ) {
+			return '<!-- error: missing VideoPress video ID -->';
+		}
+
+		$width  = (int) $atts['width'];
+		$height = (int) $atts['height'];
+		$src    = esc_url( 'https://videopress.com/embed/' . $guid );
+
+		wp_enqueue_script( 'videopress-iframe', 'https://videopress.com/videopress-iframe.js', array(), Package_Version::PACKAGE_VERSION, true );
+
+		return '<iframe src="' . $src . '" frameborder="0" allowfullscreen allow="clipboard-write" width="' . $width . '" height="' . $height . '"></iframe>';
+	}
+
+	/**
+	 * Generates a VideoPress block content with the given guid
 	 *
 	 * @param string  $content Post content.
 	 * @param WP_Post $post Post.
 	 * @return string
 	 */
-	public static function video_block_by_guid( $content, $post ) {
+	public static function videopress_block_by_guid( $content, $post ) {
```

Comment:
I wouldn't remove the `video` from there because it's the block's name. 

```suggestion
	public static function videopress_video_block_by_guid( $content, $post ) {
```